video/out/vulkan/context_display.c

```diff
@@ -471,6 +473,18 @@ static bool display_reconfig(struct ra_ctx *ctx)
 
 static int display_control(struct ra_ctx *ctx, int *events, int request, void *arg)
 {
+    struct priv *p = ctx->priv;
+    switch (request) {
+    case VOCTRL_GET_DISPLAY_FPS: {
```

Comment:
nit: No need for brackets.

Reply:
Fixed. (I copied these from similar code in `drm_common.c`)